docs/onboarding.md

```diff
@@ -0,0 +1,23 @@
+# Getting Started - Onboarding
+
+## Request Keycloak Client
+
+- create github issue (link to issue) for keycloak client creation (TBD)
+- on completetion of github issue, the user will be provided a *client_id*, *realm*, *client_secret*. All of which are required to be kept secured at all times.
```

Comment:
```suggestion
On completetion of github issue, the user will be provided [OpenId Connect](https://openid.net/connect/) information such  as *client_id*, *realm*, *client_secret*. All of which are required to be kept secured at all times.
```

Reply:
@alexjoybc can you re suggest and I'll commit.